samples/Foundatio.ReceiverConsole/App.config

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You probably don't need this file.

Reply:
We should be able to remove the app.config completely in all projects

Reply:
I have actually deleted the sample project itself. I didn't find any value to that when we have good amount of unit test cases covering all the test cases.